dashboard/src/actions/auth.ts

```diff
@@ -25,14 +25,15 @@ const allActions = [setAuthenticated, authenticating, authenticationError, setSe
 export type AuthAction = ActionType<typeof allActions[number]>;
 
 export function authenticate(
+  cluster: string,
   token: string,
   oidc: boolean,
 ): ThunkAction<Promise<void>, IStoreState, null, AuthAction> {
   return async dispatch => {
     dispatch(authenticating());
     try {
       if (!oidc) {
-        await Auth.validateToken(token);
+        await Auth.validateToken(cluster, token);
```

Comment:
what's the expected UX if someone tries to use several clusters without OIDC?

Reply:
My preference would be that we detect the issue when parsing the config (config reducer) and state that it's not supported. Right now you'd simply not be able to login. What do you think?

Reply:
we could do that at installation time? We can fail if a second cluster is given and OIDC is not enabled (https://austindewey.com/2018/12/28/helm-tricks-input-validation-with-required-and-fail/)

Reply:
Even better. Added to 1764.